build/configs/default_config.yaml

```diff
@@ -65,9 +65,24 @@ networks:
     ethereumNetworkId: 3
     # Latest deployed Smart Contracts for the given testnet
     contractAddresses:
-      identityFactory: "0x1ba18b61337fa121339339b554730c0e70b9a81c"
-      anchorRepository: "0xff5f35f6f3910ed66a24b6e55542e66b868416a3"
-      paymentObligation: "0x48c25e7639e888e667f20d30b96653de054251d8"
+      identityFactory: "0xF86177b275E4Fcb2Ab5A3FcF5E46aab51f73D7a6"
+      anchorRepository: "0xf630d8bE5664C7FB1234CEf48b6A729cEe3B71D8"
+      paymentObligation: "0x9eee8FbE3E29c96cC944F0eeC1594fe1b633dABe"
+
+  # Main net test network
+  embarcaderotest:
+    # Numeric ID of the Centrifuge network
+    id: 54
```

Comment:
Can we use here number 1 as well? Or reserve a decimal like 10?

Reply:
sure